mmv1/products/vertexai/IndexEndpointDeployedIndex.yaml

```
@@ -0,0 +1,264 @@
+    required: true
+    diff_suppress_func: 'tpgresource.ProjectNumberDiffSuppress'
+  - !ruby/object:Api::Type::String
+    name: 'displayName'
+    immutable: true
+    description: The display name of the Index. The name can be up to 128 characters long and can consist of any UTF-8 characters.
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: The timestamp of when the Index was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::NestedObject
+    name: privateEndpoints
+    output: true
+    description: Provides paths for users to send requests directly to the deployed index services running on Cloud via private services access
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'matchGrpcAddress'
+        description: The ip address used to send match gRPC requests.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'serviceAttachment'
+        description: The name of the service attachment resource. Populated if private service connect is enabled.
+        output: true
+      - !ruby/object:Api::Type::Array
+        name: 'pscAutomatedEndpoints'
+        description: |
+          PscAutomatedEndpoints is populated if private service connect is enabled if PscAutomatedConfig is set.
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: projectId
+              output: true
+              description: |
+                Corresponding projectId in pscAutomationConfigs
+            - !ruby/object:Api::Type::String
+              name: network
+              output: true
+              description: |
+                Corresponding network in pscAutomationConfigs.
+            - !ruby/object:Api::Type::String
+              name: 'matchAddress'
+              output: true
+              description: |
+                ip Address created by the automated forwarding rule.
+  - !ruby/object:Api::Type::String
+    name: 'indexSyncTime'
+    output: true
+    description: If this timestamp's value is at least the Index.update_time of the original Index, it means that this DeployedIndex and the original Index are in sync. If this timestamp is older, then to see which updates this DeployedIndex already contains (and which it does not), one must list the operations that are running on the original Index.
+  - !ruby/object:Api::Type::NestedObject
+    name: automaticResources
+    description: A description of resources that the DeployedIndex uses, which to large degree are decided by Vertex AI, and optionally allows only a modest additional configuration.
+    default_from_api: true
+    properties:
+      - !ruby/object:Api::Type::Integer
+        name: 'minReplicaCount'
+        description: The minimum number of replicas this DeployedModel will be always deployed on.
```

```suggestion
        description: |
                The minimum number of replicas this DeployedModel will be always deployed on. If minReplicaCount is not set, the default value is 2 (we don't provide SLA when minReplicaCount=1).
        
                If traffic against it increases, it may dynamically be deployed onto more replicas up to [maxReplicaCount](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/AutomaticResources#FIELDS.max_replica_count), and as traffic decreases, some of these extra replicas may be freed. If the requested value is too large, the deployment will error.
```

Adding in info from [the description of the parent field](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.indexEndpoints#DeployedIndex.FIELDS.automatic_resources) and [the field's description](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/AutomaticResources)